### PR TITLE
MBL: Add magnet compensation parameter

### PR DESCRIPTION
**Pull request by sarusani**
If the user selected 3x3 in the LCD menu he can't set if he wants to use magnet compensation or not. (It's only available for 7x7)
So I added a magnet compensation parameter (M). It is only in use if a 7x7 mesh bed leveling is performed.

### CODE REVIEW

File: Firmware/Marlin_main.cpp [hunk 1]
```diff
@@ -4882,12 +4883,13 @@ void process_commands()
...
     #### Usage
 	  
-          G80 [ N | C | O | L | R | F | B | X | Y | W | H ]
+          G80 [ N | C | O | M | L | R | F | B | X | Y | W | H ]
       
 	#### Parameters
       - `N` - Number of mesh points on x axis. Default is 3. Valid values are 3 and 7.
       - `C` - Probe retry counts. Default 3 max. 10
       - `O` - Return to origin. Default 1 (true)
+      - `M` - Use magnet compensation. Will only be used if number of mesh points is set to 7. Default is value stored in EEPROM. Vaild values are 0 (false) and 1 (ture).
```
Comment (gudnimg) [inline Firmware/Marlin_main.cpp]: Minor nitpick:  `Vaild` => `Valid`

**Review by gudnimg — APPROVED** [review]

**Review by gudnimg — APPROVED** [review]

**Review by leptun — APPROVED** [review]
Lgtm, but I'm a bit on the fence about choosing `M` as the argument letter. Don't really like that part, but it should work now that only the first letter in the command is interpreted as the "command", so `M` is properly identified as an argument.

**Comment by sarusani** [comment]
I feel the same way. I just wasn't able to come up with something else that remotely makes sense and doesn't clash with the other arguments. I was thinking about E for "elimination". But outside of the code, the feature is always referred to as "magnet compensation".

**Comment by 3d-gussner** [comment]
@sarusani Had to fix the merge conflict as I merged 4391 first.

**Pull request merged**